Sources/ComposableArchitecture/Effects/Throttle.swift

```diff
@@ -1,8 +1,8 @@
-import Combine
+@preconcurrency import Combine
 import Dispatch
 import Foundation
 
-extension Effect {
+extension Effect where Action: Sendable {
```

Comment:
Because the action flows through Combine across potential concurrency boundaries, I think we're forced to require a sendable action here. It's possible we could weaken this with a pure Swift concurrency throttle.